corehq/apps/app_manager/models.py

```diff
@@ -2705,6 +2705,13 @@ def new_form(self, name, lang, attachment=Ellipsis):
         form = self.get_form(-1)
         if attachment == Ellipsis:
             attachment = get_blank_form_xml(name)
+        elif attachment == "default_followup_form":
+            msg = "This is your follow up form. Delete this label and add questions for any follow up visits."
+            context = {
+                'lang': lang,
+                'default_label': msg
+            }
+            attachment = get_default_followup_form_xml(name, context)
```

Comment:
this looks good, but you might want to consider adding a test to prevent the functionality from getting broken in the future.

Reply:
Agreed - might actually be nice to have a test that creates a new app, adds a case list with this form, and then does a build. If this breaks in the future, it's going to be very visible. Nick, does that sound reasonable? And if so, can you help get Gabriella oriented to app manager tests and how to do it?